src/components/RoomNameInput.js

```diff
@@ -108,7 +96,7 @@ class RoomNameInput extends Component {
                 prefixCharacter={CONST.POLICY.ROOM_PREFIX}
                 placeholder={this.props.translate('newRoomPage.social')}
                 onChange={this.setModifiedRoomName}
-                value={this.state.roomName.substring(1)}
+                defaultValue={this.props.initialValue.substring(1)}
```

Comment:
Can you please add a comment here explaining why the `substring()` is needed?

Reply:
> Can you please add a comment here explaining why the `substring()` is needed?

Done